pkg/encryption/examples_test.go

```diff
@@ -27,7 +27,7 @@ func ExampleEncryptPath() {
 		panic(err)
 	}
 	fmt.Println("path to encrypt:", path)
-	fmt.Println("encrypted path: ", encryptedPath)
+	fmt.Println("encrypted path: ", []byte(encryptedPath))
```

Comment:
Can we do `hex.EncodeToString` or something instead of `[]byte` so that the output is more human readable?

Reply:
use `hex.EncodeToString` for now